api/envoy/config/rbac/v2/rbac.proto

```diff
@@ -204,6 +204,11 @@ message Principal {
     // A CIDR block that describes the downstream IP.
```

Comment:
Given the new remote_ip field, please clarify the docs for this option. In other places in envoy (access loggers, code interfaces), this is referred to as the "direct" remote address.